generator/text_pat_generators.py

```diff
@@ -0,0 +1,55 @@
+import random
+import string
+from scipy.stats import geom
+
+class uniform_generator:
+    def generate(text_len, pat_len, alph = string.ascii_lowercase):
```

Comment:
n, m, A = string.ascii_lowercase

I guess everywhere in the repository A stands for alphabet, t and n for text and its length, w and m for pattern and its length